src/backends/backend/triton_model.cc

```diff
@@ -527,6 +527,33 @@ TRITONBACKEND_RequestCorrelationId(TRITONBACKEND_Request* request, uint64_t* id)
   return nullptr;  // success
 }
 
+TRITONSERVER_Error*
+TRITONBACKEND_RequestStartsSequence(
+    TRITONBACKEND_Request* request, bool* starts_seq)
+{
+  InferenceRequest* tr = reinterpret_cast<InferenceRequest*>(request);
+  *starts_seq = tr->Flags() & TRITONSERVER_REQUEST_FLAG_SEQUENCE_START;
+  return nullptr;  // success
+}
+
+TRITONSERVER_Error*
+TRITONBACKEND_RequestEndsSequence(
+    TRITONBACKEND_Request* request, bool* ends_seq)
+{
+  InferenceRequest* tr = reinterpret_cast<InferenceRequest*>(request);
+  *ends_seq = tr->Flags() & TRITONSERVER_REQUEST_FLAG_SEQUENCE_END;
+  return nullptr;  // success
+}
+
+TRITONSERVER_Error*
+TRITONBACKEND_RequestBatchSize(
```

Comment:
We are not exposing batch-size explicitly anymore. InferenceRequest::BatchSize is going to be removed once all backends are moved over to the new API.